staging/src/k8s.io/legacy-cloud-providers/vsphere/vclib/connection_test.go

```diff
@@ -83,14 +82,14 @@ func createTestServer(
 func TestWithValidCaCert(t *testing.T) {
 	handler, verifyConnectionWasMade := getRequestVerifier(t)
 
-	server, _ := createTestServer(t, fixtures.CaCertPath, fixtures.ServerCertPath, fixtures.ServerKeyPath, handler)
+	server, _ := createTestServer(t, "./testdata/ca.pem", "./testdata/server.pem", "./testdata/server.key", handler)
```

Comment:
this isn't really bazel-related, right? worth doing separately?

Reply:
You found it already but for future reviewers https://github.com/kubernetes/kubernetes/pull/99923#discussion_r589101992

It could be done seperately.